appveyor.yml

```diff
@@ -102,7 +100,7 @@ test_script:
             }
         }
     }
-    if ($testErrorsTotal -ne 0  -and  $env:IGNORE_TEST_RESULT -ne "true") {
```

Comment:
Do you want to remove this variable or just for testing -- WIP?

Reply:
I've been mulling over it over night.

I want to remove all of this plumbing for the following reasons:
- Both master and release/*  branches are release branches, and shouldn't contain any "for testing purposes" infra. The latter belongs to pull-requests/feature branches.
- We must fail a build if either unit or integration tests fail. If unit tests fail - we have regressed, it must be fixed, no point running integration tests.

If you need this plumbing for testing purposes, please push these changes to branches on the main trunk instead of your fork. This way changes to appveyor.yml will be executed.